### ISSUE TITLE
Spigot 1.14 gamemode time cycle bug

### ISSUE THREAD

**Issue by Izaic** [EssentialsX/Essentials#2516]
### Information

**Full output of `/ess version`**:

```
[20:45:47] [Server thread/INFO]: CONSOLE issued server command: /ess version
[20:45:47] [Server thread/INFO]: Server version: 1.14-R0.1-SNAPSHOT git-Spigot-1eece4f-5a37f43 (MC: 1.14)
[20:45:47] [Server thread/INFO]: EssentialsX version: 2.16.1.154
[20:45:47] [Server thread/INFO]: Vault is not installed. Chat and permissions may not work.
[20:45:47] [Server thread/INFO]: You are running an unsupported server version!
>
```

**Logs**: https://gist.github.com/Izaic/0ad27f0d91f5e3a92aec53c0d4c5b59f

### Details

**Description**  
When night comes and you use the gamemode command, the time resets to day.

**Expected behavior**  
Time to stay the same

**Screenshots**  
![image](https://user-images.githubusercontent.com/31933482/57001966-bd2c4f00-6b89-11e9-8bc6-c233521dc085.png)

**Comment by Kongolan** [comment]
It's true, happened to me unregulary too, but only on an empty server. 1.14 really needs fixes, it's close to be stable, but Essentials is a major crash factor right now!!

**Comment by mickplouffe** [comment]
Confirm for me For me: The time resets to day when: 
Entering AFK
Existing AFK
Someone Die
Someone Join the server
...

**Comment by mdcfe** [comment]
This seems to be the result of a change in CB or Spigot, as there's nothing in EssentialsX that should be changing the time.

@Kongolan Neither Spigot 1.14 nor Paper 1.14 are anywhere near close to working properly yet - not even vanilla 1.14 works properly. EssentialsX is not a "major crash factor" in comparison to the 1.14 server software. Wait for 1.14.1 support and for fixes from Spigot and Paper.

**Comment by Izaic** [comment]
> This seems to be the result of a change in CB or Spigot, as there's nothing in EssentialsX that should be changing the time.
> 
> @Kongolan Neither Spigot 1.14 nor Paper 1.14 are anywhere near close to working properly yet - not even vanilla 1.14 works properly. EssentialsX is not a "major crash factor" in comparison to the 1.14 server software. Wait for 1.14.1 support and for fixes from Spigot and Paper.

It's been solved in 1.14.1
